package.json

```diff
@@ -92,7 +93,7 @@
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "8.0.4",
     "react-native-plaid-link-sdk": "^7.1.0",
-    "react-native-reanimated": "^2.3.0-alpha.1",
+    "react-native-reanimated": "^2.3.0-beta.3",
```

Comment:
Hey @Beamanator @parasharrajat this change here seems to have broken the Attachment Picker for me on iOS: https://expensify.slack.com/archives/C01GTK53T8Q/p1635874753343600

Reply:
Hey, @kidroca this is PR is reverted now. Thanks for notifying me.